SharedPackages/AttributedMetric/Sources/AttributedMetric/AttributedMetricPixel.swift

```diff
@@ -20,6 +20,19 @@ import Foundation
 import PixelKit
 import Common
 
+enum AttributedMetricPixelName: String {
```

Comment:
- Name extracted from the pixel so can be used as key for identifying the pixel data in the storage.
- bucket version param added